controllers/fluxcd/application-controller.go

```diff
@@ -492,6 +493,13 @@ func convertKubeconfig(kubeconfig *helmv2.KubeConfig) *kusv1.KubeConfig {
 	return &kusv1.KubeConfig{SecretRef: kubeconfig.SecretRef}
 }
 
+func convertInterval(interval *metav1.Duration) metav1.Duration {
+	if interval == nil {
+		return metav1.Duration{Duration: 10 * time.Minute}
```

Comment:
Please feel free to add unit tests, especially for this line.

Reply:
Thanks, will do